backend/rebutify/settings.py

```diff
@@ -211,3 +212,7 @@
     "SEND_CONFIRMATION_EMAIL": True,
     "SERIALIZERS": {},
 }
+
+SIMPLE_JWT = {
+    "AUTH_HEADER_TYPES": ("JWT",),
```

Comment:
By default, many JWT libraries look for a **Bearer** token in the ``Authorization`` header, i.e., ``Authorization: Bearer <token>``.
In this configuration, it is set to look for a **JWT token** instead, i.e., ``Authorization: JWT <token>``.  

So this needs to be documented for the front-end API usage.